gapis/api/vulkan/api/enums.api

```diff
@@ -281,6 +281,9 @@ enum VkStructureType {
   VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_MAINTENANCE_3_PROPERTIES              = 1000168000,
   VK_STRUCTURE_TYPE_DESCRIPTOR_SET_LAYOUT_SUPPORT                         = 1000168001,
   VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_SHADER_DRAW_PARAMETER_FEATURES        = 1000063000,
+
+  // GAPID virtual swapchain
+  VK_STRUCTURE_TYPE_VIRTUAL_SWAPCHAIN = 0xFFFFFFAA,
```

Comment:
We probably shouldn't add this to the API file.

Reply:
Removed in f766cf3